src/core/sys/posix/sys/stat.d

```diff
@@ -1294,11 +1294,11 @@ else version (CRuntime_Musl)
         timespec st_atim;
         timespec st_mtim;
         timespec st_ctim;
-        extern(D)
+        extern(D) @safe @property
```

Comment:
I hit this too, wonder why `@safe` is only needed with this C runtime.

Reply:
Not sure. Other C runtime doesn't use this definition. e.g. in CRuntime_Glibc this is behind a `(__USE_MISC || __USE_XOPEN2KB)` check, which is false.